images/win/scripts/Installers/Install-RootCA.ps1

```diff
@@ -104,6 +104,27 @@ function Disable-RootAutoUpdate {
     Set-ItemProperty $regPath -Name $regKey -Type DWord -Value 1
 }
 
+function Invoke-WithRetry {
+     param([ScriptBlock]$Command, [ScriptBlock] $BreakCondition, [int] $RetryCount=4, [int] $Sleep=1)
```

Comment:
Increase sleep time to 10 seconds and retry count to 5 by default

Reply:
done